### DISCUSSION TITLE
Is there a way to extract the version of `swc_core` for each releases?

### DISCUSSION THREAD

**Discussion by kdy1** [farm-fe/farm#1744] [General]
Hi. I'm the author of the SWC project, and I'm working to create a compatibility table for SWC Wasm plugins.

https://plugins.swc.rs/compat/range/16

<img width="1840" alt="image" src="https://github.com/user-attachments/assets/be0df730-fac0-412f-a02b-2248d4908d93">

This is what it looks like, and I want to add farm to the runtime list (assuming that farm supports Wasm plugins).
But [the import script](https://github.com/dudykr/oss/blob/16f84b868ae79f8c237ef96b4102393c19251ac0/apps/swc-plugins/scripts/import-runtime.mjs) is quite naive and I want to ask how should I parse tags.

Is checking for tags with `@farm/core` reliable?

**Comment by wre232114** [comment]
Farm supports swc wasm plugins, but farm does not use `swc_core`, it uses `swc_plugin_runner` directly https://github.com/farm-fe/farm/blob/main/crates/plugin_script/Cargo.toml#L21.

**Reply by kdy1** [reply]
Thank you!
